manager.go

```diff
@@ -95,6 +96,35 @@ func (m *Manager) LoadConfig(cfgPath string) error {
 	return nil
 }
 
+var ErrMissingCloudCreds = errors.New("can't create NetAppender without cloud creds")
```

Comment:
I'm not sure why this error is special (and gets a variable) when the other two similar checks below just directly `return errors.New(...)`

Reply:
removed in 5312398

I think I was planning to detect this + fail differently? but not sure, removed